packages/data-loader/src/commands/import.ts

```diff
@@ -71,6 +66,26 @@ export const builder = (yargs: any) =>
     .option("pfx-file-password", {
       describe: "The password of client certificate file",
       type: "string",
-    });
+    })
+    // NOTE: Unexpected `undefined` in inferred type by `option()`.
```

Comment:
It'd be more helpful if we can understand that it's just a workaround, and incorrect type inferrence was caused by yargs.
So... what about something like this?

```suggestion
    // NOTE: Since yargs doesn't detect the type correctly by adding `demandOption: true` in `option()`,
    // (inferred type always contains `| undefined`)
    // we declare the required params later as a workaround.
```